### PR TITLE
add skeleton state to composer

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes
Add Skeleton State fallback to NFT Composer on mobile app

### Demo or Before/After Pics
Before
https://github.com/gallery-so/gallery/assets/80802871/c955ceb1-6e81-45ce-ac8e-354032e2e9ba

After
https://github.com/gallery-so/gallery/assets/80802871/55690f43-66af-4ff4-b973-89624f0031e2

### Edge Cases
na

### Testing Steps
popen post composer

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/screens/PostScreen/PostComposerScreen.tsx [hunk 1]
```diff
@@ -313,7 +316,7 @@ export function PostComposerScreen() {
       eventContext={null}
     >
       <View className="flex-1 bg-offWhite dark:bg-black-900" style={{ paddingTop: top }}>
-        <Suspense fallback={null}>
+        <Suspense fallback={<PostComposerScreenSkeleton />}>
```
Comment (jakzaizzat) [inline apps/mobile/src/screens/PostScreen/PostComposerScreen.tsx:319]: This fallback is big brain 🧠

**Review by jakzaizzat — APPROVED** [review]
